packages/navie/src/user-context.ts

```diff
@@ -0,0 +1,69 @@
+export namespace UserContext {
+  export type CodeSelectionItem = {
+    type: 'code-selection';
+    location: string;
```

Comment:
Since we know this is based on a file, that file can be a candidate for code generation output.